src/strings/PowerFxResources.en-US.resx

```diff
@@ -4694,4 +4694,24 @@
     <value>The specified data source cannot be used with this function.</value>
     <comment>Error Message.</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="ErrDivByZero" xml:space="preserve">
```

Comment:
We should think about splitting the runtime resources from the "compile-time" ones. The latter are used in all platforms during "compilation", but the former are only used in those platforms that use the interpreter, so it will be a perf hit to add additional "runtime" resources that will not be used.